bindings/node.js/kzg.cxx

```diff
@@ -225,11 +219,6 @@ Napi::Value BlobToKzgCommitment(const Napi::CallbackInfo& info) {
  */
 Napi::Value ComputeKzgProof(const Napi::CallbackInfo& info) {
   Napi::Env env = info.Env();
-  size_t argument_count = info.Length();
```

Comment:
I don't fully understand what allows us to ignore all these arg length checks now. What changed?

Reply:
This is not something I feel strongly about but its a general best-practice for JS.  It relates to "duck typing" and when applied to this circumstance for function arguments.   Generally, if an argument is unused it is ignored.  In our case the arguments that come after the `argument_count` are unused, and thus could just be ignored instead of throwing the error.  I will be happy to put it back in though if there is concern that the API will be different for the node bindings

Reply:
I'm more concerned about providing less arguments than expected. For example, if 2 arguments are provided when 3 are expected, we will dig into `info[2]` which is out-of-bounds.

Are we protecting against this?

Reply:
Yep @asn-d6.  We fully vet each argument that gets used.   As an example if a `Bytes32` is expected in the second parameter position, that argument position, (`info[1]`), is checked for existence, for the expected type, `Uint8Array`, and then `.ByteLength()` is called to check it matches 32 bytes.  A distinct error gets thrown in the JS environment for each of those circumstances.  All of the above holds true for every value that gets passed from JS to C for use by the library.

Its only for extra arguments (out-of-bounds) that I was thinking about, like:
```typescript
const isUnused = true;
const result = verifyKzgProof(commitment, z_bytes, y_bytes, proof, isUnused);
```
TypeScript would hail an error for that but in regular JS it would be a valid call.